projects/demo/src/app/demo/pages/chip-page/chip-page.component.html

```diff
@@ -0,0 +1,60 @@
+<pa-demo-page>
+    <pa-demo-title>Chip</pa-demo-title>
+    <pa-demo-description>Small and compact chip used to display information, make selections, filter content or input visual content.
```

Comment:
too much unrelated stuff: it's used to display information like text and avatar, and it contain a close/clear button. 
But by itself, this component doesn't make selections, filter or input ;)

Reply:
can be "used" to "make selections"... these are basic usages of chips... I can change it :)